rj_gameplay/rj_gameplay/gameplay_node.py

```diff
@@ -1,46 +1,62 @@
+"""
+ROS entry point into Python-side gameplay library.
+Alternatively, where gameplay cedes control to cpp motion control/planning.
+
+Contains TestPlaySelector, GameplayNode, and main() which spins GameplayNode
+and allows the PlaySelector to be changed between Test and other forms.
+"""
+
+from typing import List, Optional, Tuple
```

Comment:
Added these typings. Also, style complained about no docstring in this file so I added one.